### PR TITLE
Add Stessa

### PR DESCRIPTION
**Pull request by 40504**

### CODE REVIEW

File: entries/s/stessa.com.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "Stessa": {
+    "domain": "stessa.com",
+    "tfa": [
+      "totp"
+    ],
+    "documentation": "https://support.stessa.com/en/articles/4649760",
```
Comment (ApCoder123) [inline entries/s/stessa.com.json]:
```suggestion
    "documentation": "https://support.stessa.com/en/articles/4649760#enabling-two-factor-authentication",
```

**Review by ApCoder123 — CHANGES_REQUESTED** [review]

**Review by ApCoder123 — APPROVED** [review]
This may also fit into "finance", but I'm not totally sure yet.

**Comment by 40504** [comment]
When I'm dubious about category choice, I always use `other` :)

**Comment by phallobst** [comment]
How about `investing`?

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**